### DISCUSSION TITLE
UART2 lost Data

### DISCUSSION THREAD

**Discussion by TS6000** [espressif/arduino-esp32#6424] [General]
Hello
    I have big problems on UART2.
    Since the change from 1.xx to ESP32 Arduino 2.0.2 based on ESP-IDF 4.4-beta1 data is lost.
    I work with Arduino Studio.

    I use UART2 with the pins (GPIO) 12 and 13

    Torsten

**Comment by Jason2866** [comment]
Please use latest Arduino Git version. Many big changes since version 2.0.2 in UART part!

**Reply by TS6000** [reply]
Since the change from 1.xx to ESP32 Arduino 2.0.2 based on ESP-IDF 4.4-beta1

**Comment by Jason2866** [comment]
Mandantory change from 2.0.2 to actual git version. No one will search errors in outdated 2.0.2 version.

**Reply by TS6000** [reply]
what is the current version if not 2.0.2

**Reply by me-no-dev** [reply]
current released version is 2.0.2, but many things have been fixed since then in master. Maybe next week we will have 2.0.3.
